README.md

```diff
@@ -48,6 +48,51 @@ For sling's management UI, you can head to http://localhost:8181/starter/index.h
 - To list all bundles and view their status: bundle:list
 - Find out why a bundle is in waiting state: diag _[bundle-id]_
 
+### Run the application using Podman
+First, install [podman](https://podman.io).
+
+Then, create a pod:
+
+```sh
+podman pod create --name pantheon-karaf -p 8181 -p 5005
+```
+
+This will create a `pantheon-karaf` pod with ports 8181 (for web access) and 5005 (for
+remote Java debugging) open.
+
+Run a mongo database container in the pod.
+
+```sh
+podman run --pod pantheon-karaf --name slingmongo -d mongo
+```
+
+Build the pantheon docker image
+
+```sh
+buildah bud --layers -f container/Dockerfile -t pantheon-karaf-app .
+```
+
+Run the sling container pod in the pod.
```

Comment:
Correct me if, i am wrong, should it be  " run the sling container in the pod."?

Reply:
You are correct. However, this mistake exists in the current README as well :). I will update this.